docs/content/docs/usage/sitemap.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How to use generate sitemap in Phenomic
+---
+
+The ``phenomicLoader`` allows you to generate sitemap very easily.
```

Comment:
Phenomic allows you to generate a sitemap very easily if you use the ``phenomicLoader``.